.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Maybe we can add the followings too:

--color
--require rails_helper

Reply:
The color config is deprecated. Regarding to requiring rails_helper here, looks like a good idea, will add it.